src/components/atoms/CyToggle.vue

```diff
@@ -0,0 +1,73 @@
+<template>
+  <label class="cy-toggle">
+    <input type="checkbox" />
+    <span class="slider round"> </span>
+  </label>
+</template>
+<script>
+export default {
+  name: 'CyToggle',
+}
+</script>
+<style scoped>
+.cy-toggle {
+  position: relative;
+  display: inline-block;
+  width: 60px;
+  height: 35px;
+}
+
+.cy-toggle input {
+  opacity: 0;
+  width: 0;
+  height: 0;
+}
+
+.slider {
+  position: absolute;
+  cursor: pointer;
+  top: 0;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  background-color: black;
+  border-style: solid;
+  border-width: 2px;
+  border-color: white;
+  -webkit-transition: 0.4s;
+  transition: 0.4s;
+}
+
+.slider:before {
+  position: absolute;
+  content: '';
+  height: 23px;
+  width: 23px;
+  left: 4px;
+  bottom: 4px;
+  background-color: white;
+  background-image: url(/cy-icon-black-on-white.svg);
+  -webkit-transition: 0.4s;
```

Comment:
Browser prefixes are not required. So please delete the `-webkit-transition` line.

Reason:
Quasar has easy integration with `postcss` as you can see in this file in our project:
https://github.com/CyCraft/cycraft.co/blob/TypographyExpose/.postcssrc.js

With PostCSS you can get a whole variety of automatic extra post-processing on our CSS, including the plugin that's already enabled that adds these browser prefixes automatically! : )